### DISCUSSION TITLE
Meow hints aren't showing up in Org mode

### DISCUSSION THREAD

**Discussion by jamesbtan** [meow-edit/meow#395] [Q&A] [answered]
I'm pretty new to Meow and Emacs in general. Meow hints show up in all my other modes except org mode. Any idea why that might be the case?

I tried searching and I didn't see any similar problems, hopefully this question is not repetitive.

**Comment by eshrh** [comment, marked as answer]
It's disabled by default because it looks bad with variable fonts and hidden stars (indent mode) which are fairly common with org mode.

https://github.com/meow-edit/meow/blob/master/FAQ.org#why-dont-the-number-hints-for-extending-selection-show-up-in-org-mode-buffers

So you'd do:

```elisp
(setq meow-expand-exclude-mode-list (remove 'org meow-expand-exclude-mode-list))
```
(you have to set the variable since the remove function returns a copy of the list without 'org)
